MAINTAINERS.md

```diff
@@ -0,0 +1,23 @@
+# Introduction
+
+The oneDPL project provides algorithms that are implemented using parallel patterns and bricks implemented
+for the various backends supported to allow oneDPL to run on CPU, GPU, and FPGA platforms. The API areas
+which a developer is a lead maintainer are listed below.
+
+
+| Name                  | Github ID           | API Area                                   |
+| --------------------- | ------------------- | ------------------------------------------ |
+| Ruslan Arutyunyan     | @rarutyun           | PSTL Offload                               |
+| Konstantin Boyarinov  | @kboyarinov         | PSTL Offload                               |
+| Pavel Dyakov          | @paveldyakov        | Random number generator APIs               |
+| Mikhail Dvorskiy      | @MikeDvorskiy       | Range-based algorithms, Sort algorithms    |
+| Adam Fidel            | @adamfidel          | Scan algorithms, Experimental Scan KT      |
+| Dan Hoeflinger        | @danhoeflinger      | Histogram, Transform, Iterators            |
+| Alexandr Konovalov    | @Alexandr-konovalov | PSTL Offload                               |
+| Sergey Kopienko       | @SergeyKopienko     | Find algorithms, Experimental Sort KT      |
```

Comment:
Tests too?

Reply:
I thought about it and didn't know if we wanted to have tests as a domain or not.  I will add it.

Reply:
@timmiesmith it's only my question, I don't know should we have tests as domain or not.

Reply:
@SergeyKopienko I think having it on the table so new contributors are aware that you are a good first contact for work on tests is a good idea.